images/win/scripts/Installers/Install-Meteor.ps1

```diff
@@ -4,3 +4,32 @@
 ################################################################################
 
 Choco-Install -PackageName meteor
+
+# Since PATH has changed, we'll reload so the current shell can use it.
+Update-SessionEnvironment
```

Comment:
I don't think this is an actual powershell command

Reply:
yep, you're right. That's also why all of the subsequent meteor commands were failing. I'll change this to use `refreshenv` which is a native powershell command. That should fix all the subsequent failures as well.